src/content/devices/input-output/js/main.js

```diff
@@ -1,10 +1,10 @@
 /*
- *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
- *
- *  Use of this source code is governed by a BSD-style license
- *  that can be found in the LICENSE file in the root of the source
- *  tree.
- */
+*  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
```

Comment:
I asked around, and I think we can leave these. Someone else more knowledgeable might disagree though :).